modules/optimizing-the-MTU-for-your-network.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// networking/network-optimization.adoc
+
+[id='optimizing-the-mtu-for-your-network-{context}']
+= Optimizing the MTU for your network
+
+There are two important maximum transmission units (MTUs): the network
+interface card (NIC) MTU and the SDN overlay's MTU.
+
+The MTU is autodetected. When the Cluster Network Operator runs, it determines
+the MTU used by the default route, then sets the MTU accordingly. If you need to
+set the MTU explicitly, then you must set the network configuration before
+installation.
+
+.Procedure
+
+To set the MTU explicitly, complete the following.
+
+. Run:
++
+----
+$ openshift-installer create installconfig
+----
+
+. If necessary, set the IP address blocks.
```

Comment:
When is this necessary?

Reply:
@Deepthidharwar When would one need to set the IP address blocks?

Reply:
Setting the IP address block is same step as setting subnet. So we can look to omit this line .